openedx/core/lib/block_structure/tests/helpers.py

```diff
@@ -147,6 +147,21 @@ def transform(self, usage_info, block_structure):
         pass
 
 
+class MockFilteringTransformer(FilteringTransformerMixin, BlockStructureTransformer):
+    """
+    A mock FilteringTransformerMixin class.
+    """
+    VERSION = 1
+
+    @classmethod
+    def name(cls):
+        # Use the class' name for Mock transformers.
+        return cls.__name__
+
+    def transform_block_filters(self, usage_info, block_structure):
+        return [block_structure.create_universal_filter()]
```

Comment:
Talked to @jcdyer offline - this is technically true for almost all classes in this module, but the cost of either:
- fixing every `MockXXX` class or
- having just one class be `StubXXX` while the others remain
is much higher than the burden of just leaving them all as-is.

Good note for future development though!